doc/cookbook/openapi3/openapi3.cabal

```diff
@@ -0,0 +1,26 @@
+name:                cookbook-openapi3
+version:             2.1
+synopsis:            OpenAPI 3.0 schema generation example
+homepage:            http://docs.servant.dev/
+license:             BSD3
+license-file:        ../../../servant/LICENSE
+author:              Servant Contributors
+maintainer:          haskell-servant-maintainers@googlegroups.com
+build-type:          Simple
+cabal-version:       >=1.10
```

Comment:
could you use 2.2 here instead ? in a similar spirit as https://github.com/haskell-servant/servant/pull/1498